contracts/src/token/link_token.cairo

```diff
@@ -74,6 +79,44 @@ mod LinkToken {
         MintableToken::permissionedBurn(account, amount)
     }
 
+    //
+    //  Upgradeable
+    //
+    #[external]
+    fn upgrade(new_class_hash: ClassHash) {
+        Ownable::assert_only_owner();
+        Upgradeable::upgrade(new_class_hash)
+    }
+
+    //
+    // Ownership
+    //
+
+    #[view]
+    fn owner() -> ContractAddress {
+        Ownable::owner()
+    }
+
+    #[view]
+    fn proposed_owner() -> ContractAddress {
+        Ownable::proposed_owner()
+    }
+
+    #[external]
+    fn transfer_ownership(new_owner: ContractAddress) {
+        Ownable::transfer_ownership(new_owner)
+    }
+
+    #[external]
+    fn accept_ownership() {
+        Ownable::accept_ownership()
+    }
+
+    #[external]
+    fn renounce_ownership() {
+        Ownable::renounce_ownership()
+    }
+
 
     //
     // ERC20
```

Comment:
There is a TODO comment at the end of this contract:

```
fn only_minter() {
        let caller = starknet::get_caller_address();
        let minter = minter();
        // todo: can we remove this check because it's already in the constructor
        assert(!minter.is_zero(), 'minter is 0');
        assert(caller == minter, 'only minter');
    }
```

Please double-check with @archseer, but I think this should be safe to remove as `caller` should never be `0` afaik.

Reply:
I left this comment originally. I also think we can remove it since the constructor already checks that minter is not zero. @archseer

Reply:
^ went ahead and removed it